v2/dtos/common/base.go

```diff
@@ -0,0 +1,22 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package common
+
+// Request defines the base content for request DTOs (data transfer objects).
+// This object and its properties correspond to the BaseRequest object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-data/2.x#/BaseRequest
+type BaseRequest struct {
+	RequestID string `json:"requestId" validate:"required,uuid"`
```

Comment:
@jpwhitemn @iain-anderson I thought the compromised way for now is not set this field as required.
what do you think?